README.md

```diff
@@ -60,6 +62,7 @@ Use "yast [command] --help" for more information about a command.
 ```bash
 ./yast search -m spiderman
 ```
+> if the movie name contains multiple words type it inside double quotes  e.g. "spiderman no way home"
```

Comment:
Do the following changes here :
1. `if` -> `If`

2.`multiple words` -> `multiple words separated by spaces`

3.`type it` -> `wrap it ` 

4. write the full command in eg.

Reply:
@qascade made a new commit. have resolved these :)